dags/socorro_import.py

```diff
@@ -93,7 +93,10 @@
     bq_dataset = "telemetry_derived"
     bq_table_name = f"{dataset}_{dataset_version}"
 
-    docker_image = "docker.io/mozilla/parquet2bigquery:20190722"
+    # This image was manually built from
+    # https://github.com/mozilla/parquet2bigquery
+    # using python:3.8.19-slim-bullseye
```

Comment:
Bumped per https://mozilla.slack.com/archives/C019WG3TTM2/p1725490574279479?thread_ts=1725488070.789599&cid=C019WG3TTM2, seems to have built correctly